WalletWasabi/KeyManagement/KeyManager.cs

```diff
@@ -798,7 +798,7 @@ public void AssertNetworkOrClearBlockState(Network expectedNetwork)
 
 		public void ClearState()
 		{
-			BlockchainState.BestHeight = 0;
+			BlockchainState.BestHeight = Height.Unknown;
```

Comment:
Why not 0? To be honest we don't really use `Height.Unknown` and I'm concerned if we start using it, it won't be handled properly.

Reply:
@lontivero commented to change it this

https://github.com/zkSNACKs/WalletWasabi/pull/2031#discussion_r316479159

Reply:
Ok, let's keep it like this.